### ISSUE TITLE
Insomnia adds invalid operationName field to GraphQL requests

### ISSUE THREAD

**Issue by mborman1** [getinsomnia/insomnia#5695]
### Expected Behavior

When I use the Insomnia "GraphQL Query" request type to create a GraphQL query, I expect Insomnia to generate a valid JSON request containing only the fields reflected in my originally constructed GraphQL request.  I expect Insomnia to recognize this request as valid and send it.

Example query:
```GraphQL
query {
    myQuery(id: "123") { 
        id
    }
}
```

Expected data sent in generated request:
```
'{"query":"query {myQuery(id: \"123\"){ id }}"}'
```

Expect a valid response from the request.

### Actual Behavior

In the query creation UI there is no indication of an invalid query. When the request is sent, Insomnia returns an error that there is an unknown operation name.

Example query (same as above):
```GraphQL
query {
    myQuery(id: "123") { 
        id
    }
}
```

Actual data sent in generated request
```
'{"query":"query {myQuery(id: \"123\"){ id }}","variables":{},"operationName":""}'
```

This causes Insomnia to return an error:
```
{
	"errors": [
		{
			"message": "Unknown operation named \"\"."
		}
	]
}
```

This behavior is undesired because the GraphQL spec does not require either `variables` or `operationName` to be sent with a valid GraphQL query. Additionally, current Insomnia docs mark these fields as optional.

### Reproduction Steps

To reproduce the error:

1. Use Insomnia's "GraphQL Query" user interface to craft a valid GraphQL query which does *not* contain a query name.
2. Send the query
3. Observe the error in the response

To see expected behavior:

1. Use Insomnia's "JSON" request user interface to craft a valid GraphQL query JSON object containing only the "query" field. `{"query":"query {myQuery(id: \"123\"){ id }}"}`
2. Send the query
3. Observe the successful query response

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Additional Information

Extending on the original issue https://github.com/Kong/insomnia/issues/5694

### Insomnia Version

2022.7.2

### What operating system are you using?

Windows

### Operating System Version

Windows 10 Enterprise version 22H2

### Installation method

web package download

### Last Known Working Insomnia version

n/a

**Comment by mborman1** [comment]
Update: The error being thrown is coming from the GraphQL provider. However, it is still caused by Insomnia's forced inclusion of the `operationName` field.

The decision to automatically include non-existent fields such as `operationName` has real ramifications based on the GraphQL specification. [See Executing Requests](https://spec.graphql.org/June2018/#sec-Executing-Requests). 

```
1. If operationName is null:
    a. If document contains exactly one operation.
        i. Return the Operation contained in the document.
    b. Otherwise produce a query error requiring operationName.
2. Otherwise:
    a. Let operation be the Operation named operationName in document.
    b. If operation was not found, produce a query error.
    c. Return operation.
```

This means, depending on how the hosting system determines "null", that sending an empty `operationName` field could either trigger `1 -> a -> i`, or trigger `2 -> b`.  Which interpretation is correct is a separate discussion, but either way Insomnia should not be imposing this field on the request objects.

**Comment by goofy78270** [comment]
after the operation name, and the first "(", you need a space.  After that, the operation name at the top needs to be corrected, possibly.
![image](https://user-images.githubusercontent.com/23242835/213030738-4d539f2f-e434-4886-af6a-5751fafcd4b5.png)

**Comment by AfrazHussain** [comment]
It took me longer than I would like to admit to actually figure out why my GQL queries weren't going through. Anyways, a fix has already been pushed with version 2022.7.4 with [this PR](https://github.com/Kong/insomnia/pull/5681). I have tested it on 2022.7.4 and can confirm that I no longer get this error.

**Comment by filfreire** [comment]
Closing this one. Fix is available from latest [2022.7.4](https://github.com/Kong/insomnia/releases/tag/core%402022.7.4) onwards.
